scripts/clustering/README.md

```diff
@@ -0,0 +1,50 @@
+# 📄 Hierarchical clustering of PCAWG samples 
+
+In order to reproduce the results of hierarchical clustering on the PCAWG samples uploaded to SVELT, please run `clustering.py`.  The aim of the tool is to organize the PCAWG samples based on the patterns of rearrangement signatures to facilitate the process of structural variants interpretation. The samples are clustered using the Euclidean metric as a distance measure.
+
+## Dependencies
+
+**Python version**: 3+
+
+**Packages**: click, pandas, matplotlib, seaborn
+
+## CLI 
+
+The help command of the tool: 
+
+```
+python3 clustering.py  --help 
+Usage: clustering.py [OPTIONS]
```

Comment:
I think you can also provide here the commands you used to generate output files. (Maybe I missed it!)